content/projects/_index.md

```diff
@@ -4,9 +4,75 @@ description: "This page provides the details of SODA Projects"
+
+    Every SODA release will have the SODA Dashboard with new use cases integrated based on the new features or projects integrated to SODA project ecosystem.
+
+- ### SODA Infrastructure Manager
+    delfin (Dolphin in Spanish!), the SODA Infrastructure Manager project is an open source project to provide unified, intelligent and scalable resource management, alert and performance monitoring. It will cover the resource management of all the storage backends & other infrastructures under SODA deployment. It will also provide the alert management and metric data(performance/health) for monitoring and further analysis.
+
+    It will provide a scalable framework where more and more backends as well as client exporters can be added. This will enable to add more storage and infrastructure backends and also support different management clients for monitoring and health prediction.
+
+    It provides unified APIs to access, export and connect with clients as well as a set of interfaces for various driver additions.
+    The vision is to build a unified intelligent and scalable infrastructure management framework for resource management (config, add, remove, update), alert management, and performance metrics management with standardized APIs across any storage and application platforms, with connectors for data analysis and prediction.
+
+- ### SODA API
+    Provides the standardization for Data / Storage Management APIs. Currently we support block and file APIs for key features of data management (provisioning, migration, fileshare, etc). Working to add the storage management APIs.
+
+    This is the key external interface to platforms, which can do a seamless integration with heterogeneous storage backends. 
+
+    Users can develop SODA North-Bound Plugins (SODA NBP) under SODA NBP project to connect any platform or application solutions to SODA API from north for all storage/data requirements.
+
+    We envision this to be the reference implementation of SODA Data Standard API Specification, which we plan to work with our industry partners and standards bodies. At that stage, this layer will upgraded to support Block, File and Object APIs across the Edge, Core and Cloud.
+
+- ### SODA Controller
+    Provides metadata management, control/configurations, scheduler and all such bookkeeping features and utilities. In the API flow from SODA API to SODA Dock, controller plays a critical role for all the API flow management and tracking to handle all the state machine and metadata management requirements. This will be a layer to keep addons to new facilities or utilities for the soda platform.
+
+    Currently we think, this layer can be optional or similar modules can be used from users(if they have such software components). However, the users needs to do certain integration with SODA API and SODA Dock for their controller modules.
```

Comment:
_However, the users needs to do certain integration with SODA API and SODA Dock for their controller modules_. 
-> 
_However, they will have to integrate their controller modules with the SODA API and SODA Dock._
We can add ` as per the integration guide` (if  we have an integration guide)